libnetwork/sandbox_windows.go

```diff
@@ -37,3 +37,8 @@ func (sb *Sandbox) populateNetworkResources(context.Context, *Endpoint) error {
 	// not implemented on Windows (Sandbox.osSbox is always nil)
 	return nil
 }
+
+func (sb *Sandbox) IPv6Enabled() (enabled, ok bool) {
+	// None of the Windows container network drivers currently support IPv6.
```

Comment:
This could be a GoDoc on the function, which may surface the information in IDEs etc. For example, my IDE (Goland) hides the function body by default in this case (hiding the comment 😬);

```go
// IPv6Enabled always returns false on Windows as None of the Windows container
// network drivers currently support IPv6.
func (sb *Sandbox) IPv6Enabled() (enabled, ok bool) {
	return false, true
}
```

Reply:
I'd never noticed GoLand hides comments like that!

Done - although now it's the only function in that file without a comment like that in its body.